src/sass/_theme_rst.sass

```diff
@@ -220,6 +220,9 @@
     background: $highlight-color
     display: inline-block
     font-weight: bold
+    font-style: italic
+    color: #404040
+    border-bottom: 2px solid rgb(255, 190, 0)
```

Comment:
Colors are already defined as variables, `#404040` should be a named color already. The yellow you have defined here is different than the yellow defined via Wyrm and should be used as a named color variable either way.

I'd also support using `#404040` as the font color for `#search-results`, so that the text in our search results is easier to read.

Reply:
If I understand correctly, wyrm defines `gray-dark` as `#333333` and `gray` as `#555555`. So, they are not exactly the same as `#404040`. What I should do?

I was thinking on adding as a variable with the `highlight-` prefix, like:

```
// Sphinx highlight color
$highlight-background-color:          rgba(255, 220, 0, 0.5)
$highlight-text-color:                #404040
```

Reply:
from `bower_components/wyrm/sass/wyrm_core/_wy_variables.sass`:

```
$black:                               #000 !default
$gray-darker:                         #222 !default
$gray-dark:                           #333 !default
$gray:                                #555 !default
$gray-light:                          #999 !default
$gray-lighter:                        #ccc !default
$white:                               #fff !default
```

Reply:
I'd probably use `$gray-dark`. The visual difference between `#333333` and `#404040` is pretty marginal.